.github/workflows/build-only.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: Build-only
+on:
+  push:
+    branches:
+      - '*'
+      - '!_*'
```

Comment:
From python's [pep8](https://peps.python.org/pep-0008/#descriptive-naming-styles): prefix with underscore implies private. 
However, I can not say that it is a widely used convention for branch name. Any other ideas?

Reply:
Ah, OK. I'd simply ask if it's needed? What are the reasons why we need a private branch to skip these tests?  Alternatively, I think that build yamls can read the labels on a PR, so maybe add a label that disables this job? Or skip it on draft?

Reply:
Private branches are used during (long) development cycle, where sometimes you may simple want to push into github as backup or sharing. In such cases, developers may know that build is broken and she wants to avoid the noise of this action failure.

Would `!*draft` be a better rule?

Reply:
Personally, I think that'd be clearer. I think a label would be even better than that, but we can do that later.